src/Test.FunctionalTests.BinSkim.Rules/RuleTests.cs

```diff
@@ -1416,6 +1416,24 @@ public void BA3030_UseCheckedFunctionsWithGCC_NotApplicable()
             this.VerifyApplicability(new UseCheckedFunctionsWithGcc(), new HashSet<string>());
         }
 
+        [Fact]
+        public void BA3031_EnableSafeStackWithClang_Pass()
```

Comment:
>public void BA3031_EnableSafeStackWithClang_Pass() [](http://example.com/codeflow?start=8&length=50)

the combination of the test files are:
1. version, from latest 14 down to 7
2. language, C and C++

(By these tests we found that lower version has different symbol name)

Reply:
Awesome, do we have build script examples to include for these?

Reply:
yes! added.